server/errorHandler.test.ts

```diff
@@ -52,3 +57,19 @@ describe('GET 404', () => {
       })
   })
 })
+
+describe('redirection on user not authorised', () => {
+  it.each([
+    [401, 'Unauthorised'],
+    [403, 'Forbidden'],
+  ])('should render not authorised page when error is %p', (status: number, message: string) => {
+    const app = setupApp(true, createError(status, message))
+
+    return request(app)
+      .get('/unauthorised')
+      .expect(302)
+      .expect(res => {
+        expect(res.text).toContain('Found. Redirecting to /not-authorised')
```

Comment:
I'm a little confused where this message comes from. I assume from somewhere in the guts of express? I'd consider changing to just `expect(res.text).toContain('Redirecting to /not-authorised')` to avoid confusion

Reply:
:recycle: